android/sdl_android/src/main/java/com/smartdevicelink/encoder/VirtualDisplayEncoder.java

```diff
@@ -107,8 +126,8 @@ public VideoStreamingParameters getStreamingParams(){
     }
 
     @SuppressWarnings("unused")
-    public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, VideoStreamingFormat format) {
-        this.streamingParams = new VideoStreamingParameters(displayDensity, frameRate, bitrate, interval, resolution, format);
+    public void setStreamingParams(int displayDensity, ImageResolution resolution, int frameRate, int bitrate, int interval, VideoStreamingFormat format, boolean stableFrameRate) {
+        this.streamingParams = new VideoStreamingParameters(displayDensity, frameRate, bitrate, interval, resolution, format, stableFrameRate);
```

Comment:
This would be considered a breaking change, can you please keep the old method and deprecate it and then add in your new method

Reply:
Thanks for the suggestion. I have restored the previous setStreamingParams method at commit #a4f2539.